torchgen/model.py

```diff
@@ -2659,6 +2663,10 @@ def gets_generated_view_copy(f: NativeFunction) -> bool:
     # We also don't need to generate copy variants for inplace views.
     if "inplace_view" in f.tags:
         return False
+    # Assume ops ending in _inverse have manually-defined copy variants
+    # (e.g. slice_inverse() has the copy variant slice_scatter())
```

Comment:
It might be helpful context to mention here why we don't just auto-gen the `slice_scatter` op (since we auto-gen every other view_copy op)

Short answer.. we probably could? (it's just cloning the base and then doing a slice into the cloned tensor). But the codegen is going to be slightly different, and making the codegen more complicated is probably more trouble than its worth compared to hand-writing ~5 kernels